uxn.h

```diff
@@ -19,124 +19,3 @@ typedef struct {
     int dst_page;
     void* dst_addr;
 } Expansion;
```

Comment:
Isn't the expansion port also part of varvara?

Reply:
For that matter, so is `exit()`.

Reply:
Thanks, I've missed those.